proposals/p0051.md

```diff
@@ -130,9 +130,9 @@ For example of where trade-offs may be seen right now:
         how could will function on various platforms.
     -   #4: Carbon's plan for `Int` trapping where C++ would overflow should
         make code safety easier.
--   We expect platform support priorities to differ between C/C++ and Carbon.
-    This is a conflict with Carbon's #6 priority, which focuses support on
-    modern over legacy platforms.
+-   We expect platform support priorities to differ between C and C++ and
+    Carbon. This is a conflict with Carbon's #6 priority, which focuses support
+    on modern over legacy platforms.
```

Comment:
```suggestion
-   We expect platform support priorities to differ between C++ and
    Carbon. This is a conflict with Carbon's #6 priority, which focuses support
    on modern over legacy platforms.
```
Platform support priorities for C versus for C++ are not the same (eg, C compilers for embedded targets are much more common than C++ compilers), and I think it's C++ that we want to contrast with here.

Reply:
Can you just confirm, you want the rationale used for this legacy proposal changed?

I'm going to say, I don't think your argument had come up at the time. As the proposal author, I'm pretty sure I meant it this way, even though you can tell me I should've said something different. I would be more comfortable not changing this, but given I know I'm in disagreement about how much legacy proposals should be changed, I just want you to confirm that you want this edit on the basis that you prefer the argument to not mention C anymore.

Reply:
Hm. I don't think there's any good edit we can make here. I wouldn't read "between C/C++ and Carbon" as meaning the same thing as "between C and C++ and Carbon". Let's leave the old text as-is.

Reply:
Restored